### DISCUSSION TITLE
The NGINX Proxy Manager never updates using the LXC updater

### DISCUSSION THREAD

**Discussion by vulcan4d** [community-scripts/ProxmoxVE#789] [General]
I spooled up the NGINX Proxy Manager LXC which runs at v2.9.20.  I then ran the LXC updater and it doesn't update.  All my other LXC containers update but this one.  Is it just not supported because I re-deployed the container and it still won't update.  It is quite out of date and considering it may be public facing that is a security risk.  Latest version is v2.12.1.  Is it just not possible doing it via the command: bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs.sh)"

**Comment by michelroegl-brunner** [comment]
There are some changes proposed for the Updater in a [PR](https://github.com/community-scripts/ProxmoxVE/pull/763)

**Comment by MickLesk** [comment]
And you use the old repo
